### DISCUSSION TITLE
TestFlight Beta

### DISCUSSION THREAD

**Discussion by atomtoto** [KG-Development/SmartBMSUtility#28] [General]
Hello, can I have a link to the TestFlight beta? Thanks!

**Comment by NeariX67** [comment]
There is no newer version available on testflight. I can update the testflight link to a usable version of V3 once it's released..

**Reply by atomtoto** [reply]
Yes, that would be cool! I also wanted to tell you that I can translate the app into French (for free of course).

**Reply by NeariX67** [reply]
Yeah, that would be cool as multiple languages will be supported in version 3!

**Reply by atomtoto** [reply]
When will the beta on TestFlight be available for download again? I was also wondering if I could have the source code of the app, so that I could try things on Xcode on my side?
Thank you!
